SharkCage/special_configs/CageConfigurator.sconfig

```diff
@@ -0,0 +1,12 @@
+{
+  "application_name": "CageConfigurator",
+  "application_path": "C:\\sharkcage\\CageConfigurator.exe",
```

Comment:
I'm currently enhancing our Installer. The `application_path` must be written by the Installer or? It depends on the target folder specified by the user while installing.